runners-modules/benchmarking.py

```diff
@@ -635,13 +636,31 @@ async def _generate_prompts_with_attack_module(self, prompt: str) -> list[str]:
         """
         # Iterate over all attack modules and apply them sequentially
         modified_prompts = []
+
```

Comment:
can we do a check on self.recipe_instance.attack_mod then do the creation and the loading

if self.recipe_instance.attack_modules:

Reply:
check is done before entering this method